deeprankcore/DataSet.py

```diff
@@ -225,44 +173,6 @@ def _check_hdf5_files(self):
         for name in remove_file:
             self.hdf5_path.remove(name)
 
-    def _check_node_feature(self):
-        """Checks if the required node features exist"""
-        f = h5py.File(self.hdf5_path[0], "r")
-        mol_key = list(f.keys())[0]
-        self.available_node_feature = list(f[f"{mol_key}/{groups.NODE}/"].keys())
-        self.available_node_feature = [key for key in self.available_node_feature if key[0] != '_'] # ignore metafeatures
-        f.close()
-
-        if self.node_feature == "all":
-            self.node_feature = self.available_node_feature
-        else:
-            for feat in self.node_feature:
-                if feat not in self.available_node_feature:
-                    _log.info(f"The node feature _{feat}_ was not found in the file {self.hdf5_path[0]}.")
-                    _log.info("\nCheck feature_modules passed to the preprocess function.\
-                        Probably, the feature wasn't generated during the preprocessing step.")
-                    _log.info(f"\nPossible node features: {self.available_node_feature}\n")
-                    sys.exit()
-
-    def _check_edge_feature(self):
-        """Checks if the required edge features exist"""
-        f = h5py.File(self.hdf5_path[0], "r")
-        mol_key = list(f.keys())[0]
-        self.available_edge_feature = list(f[f"{mol_key}/{groups.EDGE}/"].keys())
-        self.available_edge_feature = [key for key in self.available_edge_feature if key[0] != '_'] # ignore metafeatures
-        f.close()
-
-        if self.edge_feature == "all":
-            self.edge_feature = self.available_edge_feature
-        elif self.edge_feature is not None:
-            for feat in self.edge_feature:
-                if feat not in self.available_edge_feature:
-                    _log.info(f"The edge feature _{feat}_ was not found in the file {self.hdf5_path[0]}.")
-                    _log.info("\nCheck feature_modules passed to the preprocess function.\
-                        Probably, the feature wasn't generated during the preprocessing step.")
-                    _log.info(f"\nPossible edge features: {self.available_edge_feature}\n")
-                    sys.exit()
-
     def load_one_graph(self, fname, mol): # noqa
         """Loads one graph
 
```

Comment:
Made load_one_graph use all available features/targets rather than just the selected ones.
However:

1. there is no test that allows me to check whether it works as intended --> create a unit test for this
2. I am not sure what the purpose of this function is, so I am unsure whether it's intended to load all possible features/targets or just the selected ones